libs/domain/picking/src/models/picking-list.ts

```diff
@@ -7,6 +7,7 @@ export interface PickingList {
   cartNote?: string;
   createdAt: Date;
   updatedAt: Date;
+  orderReferences: string[];
```

Comment:
Why is it added on the standard model? It should not be used here as we are not supporting this search in online mode because BAPI should be doing it which it doesn't yet. But frontend model does not need this info.

Reply:
This will be cover on: https://github.com/spryker/oryx/pull/317